irispy/extern/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Do we need to be writing an extern if we don't have anything to put it to it? is it a cookie option?

Reply:
There is a c extern option but not a normal extern one.